### DISCUSSION TITLE
MDatePicker改进建议

### DISCUSSION THREAD

**Discussion by ldming-com** [BlazorComponent/MASA.Blazor#1111] [Ideas]
1. 建议拆分单日期选择和范围选择
2. 时间状态应该可以组合的（yyyy、yyyy-MM、yyyy-MM-dd、yyyy-MM-dd HH、yyyy-MM-dd HH:mm、yyyy-MM-dd HH:mm:ss）
3.像这种选择器大多数据场景都是作为文本输入框的扩展增加，现在要实现文本框聚焦时（或点击前后图标）打开还得借助Menu组件感觉使用有点繁琐。
4.这类控件定位应该是辅助，而非替代，在和文本框联合使用时最好能保留用户输入的权利。

很多时候，这类控件有时很方便，但有些场景下非常繁琐，特别是在大批量录入数据和时间跨度比较大的情况下，要点很多次才能完成。如果保留输入，配合Rules效验，在用户能正确输入时，明显比选择方便、快捷得多，而且很多时候用户在批量录入时都是从其他地方复制过来粘贴的。

参考：
https://www.blazor.zone/datetimepickers

**Comment by doddgu** [comment]
可以提个issue，晚点我们选一个符合国人使用习惯的日期组件做个JsProxy类组件即可。
或者你对这个感兴趣，也可以给我们提pr，JsProxy的上手难度会低很多，类似ECharts这样。
